### PR TITLE
testutil/integration: add DKG test with 5000 validators 

### PR DESCRIPTION
**Pull request by gsora**
Also rename longwait_dkg_test.go to nightly_dkg_test.go, since it contains more than a DKG long-wait test.

category: test
ticket: none

### CODE REVIEW

File: .github/workflows/nightly-dkg.yml [hunk 1]
```diff
@@ -1,5 +1,5 @@
-# This runs long wait DKG test once a day at 12:00 noon.
-name: long wait dkg
+# This runs DKG tests that assert edge cases and long execution times once a day at 12:00 noon.
+name: nightly dkg
```
Comment (xenowits) [inline .github/workflows/nightly-dkg.yml:2]: fun: It is more of a `day dkg` test since it happens at noon, not night 🫣

Please ignore this comment

**Review by xenowits — APPROVED** [review]
